_test/test_sqw/test_PixelData.m

```diff
@@ -597,7 +597,10 @@ function test_dirty_pix_tmp_files_are_deleted_when_pix_out_of_scope(obj)
             old_rng_state = rng();
             fixed_seed = 774015;
             rng(fixed_seed, 'twister');  % this seed gives an expected object_id_ = 54452
-            expected_tmp_dir = fullfile(tempdir(), 'sqw_pix54452');
+            expected_tmp_dir = fullfile( ...
+                get(parallel_config, 'working_directory'), ...
+                'sqw_pix54452' ...
+            );
             clean_up = onCleanup(@() rng(old_rng_state));
```

Comment:
Shouldn't the onCleanup call precede the rng call on L.599? Ditto at L.899

Reply:
They will work fine where they are: the `old_rng_state` variable is set as the rng is reseeded. This call creates function pointer that is executed at the end of the method to restore that.

Reply:
@nickbattam-tessella What if rng or fullfile errors?

Reply:
I've moved the call up 👍